ts/blueprints/ember-cli-typescript/index.js

```diff
@@ -146,8 +146,8 @@ module.exports = {
       { name: 'ember-cli-typescript-blueprints', target: 'latest' },
       { name: 'typescript', target: 'latest' },
       { name: '@types/ember', target: 'latest' },
-      { name: '@types/rsvp', target: 'latest' },
```

Comment:
`@types/ember` already brings in `@types/rsvp`, and having one fewer explicit version constraint floating around makes "cannot re-declare block scoped variables" more likely to crop up as folks update various dependencies.